bschweiz/HappyPlants
# Dec11 plant form and list

# Description
Plant Form and List modules are working with no bugs but the Plant Form completion is not integrated with the results being returned from Trefle yet
Fixes # (bugs in the pLant modules )
## Type of change
Please delete options that are not relevant.
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
Navigate to add plant and make a plant, see it on plant list. i still need to reroute on save plant button click
# Checklist:
- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
